### PR TITLE
fix(metric): use loss mask if not none

### LINKED ISSUE #1259: [Bug]: Advantage metrics should respect loss_mask
### Affected component

metrics.

### Bug description

The current rollout advantage metrics compute `advantages_mean`, `advantages_min`, and `advantages_max` over the full advantage tensor without applying loss_mask.

For embodied PPO rollouts, some positions can be invalid due to episode termination, padding, or masked action chunks. Including these invalid positions makes the logged advantage metrics inaccurate and potentially misleading, especially for min/max values.

These metrics should be computed over valid positions only when loss_mask is available.

### Configuration YAML file

YAML config file:

You can also paste the full config here.
```
<paste YAML content here>
```

### Log file

You can find the log file in `logs/` folder or the $output_dir/$experiment_name folder (defined in the yaml config) if you are using our example scripts.

Log file:

If you cannot find the log, please provide the full log messages here.
```
<paste log here>
```

### Environment

python -V:
uv pip list:
nvidia-smi:
RLinf version:
Docker image tag: 

### Other reproduction info

_No response_

### Before submitting a new issue...

- [x] Have you checked relevant issues, FAQs (https://rlinf.readthedocs.io/en/latest/rst_source/faq.html), or asked the chatbot at the top right corner of the [documentation page](https://rlinf.readthedocs.io).

### PR DESCRIPTION
**Pull request by anHappyDog**
### Description
Fix #1259, appy loss mask to adv/return/rewards for metric computation.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (Document-only update)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by andylin-hao — APPROVED** [review]

**Review by qurakchin — APPROVED** [review]

File: rlinf/utils/metric_utils.py [hunk 1]
```diff
@@ -117,60 +117,81 @@ def compute_evaluate_metrics(eval_metrics_list):
...
+            values = values.to(device)
+            count = torch.tensor(
+                values.numel(), device=values.device, dtype=torch.float64
+            )
+            values_sum = values.to(dtype=torch.float64).sum()
+            max_value = torch.max(values).detach().item()
+            min_value = torch.min(values).detach().item()
+
+        reduce_sum_count = torch.stack([values_sum, count])
+        reduce_tensor = torch.as_tensor(
```
Comment (thereAreDemonsNearby) [inline rlinf/utils/metric_utils.py]: Should this variable be named **reduce_max_tensor**, since you have a **reduce_sum_count**?

Comment (anHappyDog) [reply rlinf/utils/metric_utils.py]: maybe `reduced_min_max` is better, I'll change it.

Comment (anHappyDog) [reply rlinf/utils/metric_utils.py]: done.

**Review by thereAreDemonsNearby — APPROVED** [review]

**Review by guozhen1997 — APPROVED** [review]
